garak/probes/topic.py

```diff
@@ -92,7 +92,8 @@ def _gen_prompts(self, term):
     def __init__(self, config_root=_config):
         super().__init__(config_root)
 
-        self.data_dir = _config.transient.cache_dir / "resources" / "wn"
+        self.data_dir = _config.transient.cache_dir / "data" / "wn"
+        self.data_dir.parent.mkdir(mode=0o740, parents=True, exist_ok=True)
```

Comment:
Interesting item here, the `parent` must exist before download however if the `wn` folder already exists the raised when attempting access below will be a different exception type and fails to recover.

This was `masked` previously by the fact that `_config.transient.cache_dir / "resources"` is created in early initialization for the `plugin_cache.json` file.

Reply:
> Where do we want to check for / make XDG dirs?
https://github.com/leondz/garak/blob/0e1d96e3892e14564aaece56c1ee9f92cfdd7dc0/garak/_config.py#L68-L71

`parents=True` is the equivalent to `-p`.

Reply:
Thanks, missed that. This looks good. Expect to move `wn` code to its own module/class once we know where those will live - did we settle on `resources`? Other options could be `apis`, `external`, `apps`, `common`, & _not_ `lib`. Or is this a conversation to conclude once this PR lands?